### PR TITLE
Ignore Header

### PR DESCRIPTION
**Pull request by ahmdrz**

### CODE REVIEW

File: mina.go [hunk 1]
```diff
@@ -116,6 +117,18 @@ func (m *Mina) ServeHTTP(wr http.ResponseWriter, req *http.Request) {
 	req.Header.Del("If-Modified-Since")
 	req.Header.Del("If-None-Match")
 
+	if req.Header.Get(XHeaderName) == XHeaderValueIgnore {
```
Comment (siadat) [inline mina.go]: Cool! Let's use a different header name for requests (e.g., RequestOptionsHeaderName being "X-MINA-OPTIONS"), and leave XHeaderName for the response header.

If you want, you could rename XHeaderName to something better as well, e.g., ResponseCacheHeaderName. :game_die:

File: mina_test.go [hunk 2]
```diff
@@ -50,6 +50,51 @@ func TestMina(t *testing.T) {
...
+	frontend := httptest.NewServer(m)
+	defer frontend.Close()
+
+	req, err := http.NewRequest("GET", frontend.URL, nil)
+	if err != nil {
+		log.Fatal(err)
+	}
+	req.Header.Add(XHeaderName, XHeaderValueIgnore)
+	client := http.Client{}
+	res, err := client.Do(req)
```
Comment (siadat) [inline mina_test.go]: Let's call the Do() method twice and then check for XHeaderValueMiss header value.

File: mina.go [hunk 3]
```diff
@@ -116,6 +117,18 @@ func (m *Mina) ServeHTTP(wr http.ResponseWriter, req *http.Request) {
...
+		wrRecorder := httptest.NewRecorder()
+		p.ServeHTTP(wrRecorder, req)
+
+		resp := wrRecorder.Result()
+		defer resp.Body.Close()
+
+		writeHeadersToWR(wr, resp, m.Headers, XHeaderValueIgnore)
```
Comment (siadat) [inline mina.go]: Let's keep the number of possible header values small, just write XHeaderValueMiss instead.

**Comment by ahmdrz** [comment]
Thank you , I will change the code and commit again

**Comment by ahmdrz** [comment]
Sorry :sweat_smile: , Long time no changes.

**Comment by siadat** [comment]
@ahmdrz thanks, good to see you back! :+1: :man_dancing:

**Pull request merged**